packages/react-component-library/package.json

```diff
@@ -157,6 +158,7 @@
     "react-select": "^3.0.8",
     "react-tether": "^2.0.7",
     "react-toast-notifications": "^2.4.0",
-    "react-transition-group": "^4.3.0"
+    "react-transition-group": "^4.3.0",
+    "styled-components": "^5.2.0"
```

Comment:
FYI https://styled-components.com/docs/faqs#i-am-a-library-author-should-i-bundle-styledcomponents-with-my-library

Reply:
Thanks for this - change has now been made - i've created a ticket off the back of this because I think there are probably some other peerDependencies that need moving out of the bundle: https://github.com/Royal-Navy/design-system/issues/1478